mujinvisioncontrollerclient/visioncontrollerclient.py

```diff
@@ -452,17 +468,8 @@ def GetVisionStatistics(self, taskId=None, cycleIndex=None, taskType=None, timeo
             command['taskType'] = taskType
         return self._ExecuteCommand(command, timeout=timeout)
     
-    def _SendConfiguration(self, configuration, fireandforget=False, timeout=2.0, checkpreempt=True, recvjson=True):
-        # type: (typing.Dict, bool, float, bool) -> typing.Dict
-        if self._callerid:
-            configuration['callerid'] = self._callerid
-        response = self._configurationsocket.SendCommand(configuration, fireandforget=fireandforget, timeout=timeout, checkpreempt=checkpreempt)
-        if not fireandforget:
-            return self._ProcessResponse(response, command=configuration, recvjson=recvjson)
-        return response
-    
     def Ping(self, timeout=2.0):
-        # type: (float) -> typing.Dict
+        # type: (float) -> Optional[Dict]
         return self._SendConfiguration({"command": "Ping"}, timeout=timeout)
     
     def SetLogLevel(self, componentLevels, timeout=2.0):
```

Comment:
no type definition?

Reply:
Not for now. When we get to it we'll do it right.

Reply:
I left the existing type hints in here, but the rest will come when they're generated